### PR TITLE
ZK-4020: Grid creates more row than expected

### PR DESCRIPTION
**Pull request by klyvechen**

### CODE REVIEW

**Comment by wenninghsu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

File: zkdoc/release-note [hunk 1]
```diff
@@ -57,6 +57,7 @@ Aug 28, 2018
   ZK-4030: popup loading overlay in wrong position
   ZK-4001: new feature title on collapse borderlayout region missing style in some themes
   ZK-4039: a caption inside a borderlayout causes a javascript error
+  ZK-4020: Grid creates more row than expected
```
Comment (wenninghsu) [inline zkdoc/release-note]: 8.6.0

**Comment by DevChu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**